extractor/build.gradle

```diff
@@ -9,4 +9,6 @@ dependencies {
 
     testImplementation 'junit:junit:4.13'
     testImplementation "com.squareup.okhttp3:okhttp:3.12.11"
+    testImplementation 'com.google.code.gson:gson:2.8.6'
```

Comment:
Are there limitations with `nanojson`?

Reply:
I don't quite remember what it was, but it did not work with nanojson

Reply:
Mmmh ok, this is anyway `testImplementation` so no need to bother about it ;-)

Reply:
My initial idea behind saving it as json (human readable) was to be able to search through it. Because i remember that when i first made the mix extractor i saved the request with postman to do just that.

If that proves to not be needed, then `ObjectInputStream` `ObjectOutputStream` can be used.

Reply:
No, we can stick to json, human readability is useful